### PR TITLE
chmod: add Spanish translation

### PR DESCRIPTION
**Pull request by GustavoHiguera**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Review by dagp-ucol — APPROVED** [review]

**Review by navarroaxel — COMMENTED** [review]
Hi @GustavoHiguera, welcome and thanks for your submission. I noticed your translation is not in sync with the main English page here: https://github.com/tldr-pages/tldr/blob/main/pages/common/chmod.md

Could you please update this PR?

**Comment by GustavoHiguera** [comment]
> Hi @GustavoHiguera, welcome and thanks for your submission. I noticed your translation is not in sync with the main English page here: https://github.com/tldr-pages/tldr/blob/main/pages/common/chmod.md
> 
> Could you please update this PR?

Sure! Thank you

File: pages.es/linux/chmod.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# chmod
+
+> Cambia los permisos de acceso de un archivo o directorio. Más información: https://www.gnu.org/software/coreutils/chmod.
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:3]:
```suggestion
> Cambia los permisos de acceso de un archivo o directorio.
> Más información: <https://www.gnu.org/software/coreutils/chmod>.
```

File: pages.es/linux/chmod.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+- Dar al usuario que posee el archivo el permiso de ejecución:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:5]:
```suggestion
- Da al [u]suario que posee al archivo el permiso de ejecución [x]:
```

File: pages.es/linux/chmod.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod u+x {{ruta/al/archivo}}`
+
+- Dar al usuario permisos de lectura y escritura al archivo o directorio:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:9]:
```suggestion
- Da al [u]suario permisos de lectu[r]a y escritura [w] a un archivo o directorio:
```

File: pages.es/linux/chmod.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod u+rw {{ruta/al/archivo_o_directorio}}`
+
+- Remover permisos de ejecución para el grupo:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:13]:
```suggestion
- Remueve permisos de ejecución [x] del [g]rupo:
```

File: pages.es/linux/chmod.md [hunk 5]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod g-x {{ruta/al/archivo}}`
+
+- Dar a todos los usuarios permisos de lectura y ejecución:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:17]:
```suggestion
- Da a todos [a] los usuarios permisos de lectu[r]a y ejecución [w]:
```

File: pages.es/linux/chmod.md [hunk 6]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod a+rx {{ruta/al/archivo}}`
+
+- Dar a otros usuarios los mismos permisos que el grupo:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:21]:
```suggestion
- Da a [o]tros usuarios (no dentro del grupo dueño del archivo) los mismos permisos que el [g]rupo:
```

File: pages.es/linux/chmod.md [hunk 7]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod o=g {{ruta/al/archivo}}`
+
+- Remover todos los permisos de los otros:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:25]:
```suggestion
- Remueve todos los permisos de [o]tros:
```

File: pages.es/linux/chmod.md [hunk 8]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod o= {{ruta/al/archivo}}`
+
+- Cambiar los permisos recursivamente dando al grupo y los otros la habilidad de escribir:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:29]:
```suggestion
- Cambia los permisos recursivamente dando al [g]rupo y a [o]tros la habilidad de escribir [w]:
```

File: pages.es/linux/chmod.md [hunk 9]
```diff
@@ -0,0 +1,35 @@
...
+
+`chmod -R g+w,o+w {{ruta/al/directorio}}`
+
+- Recursivamente dar a todos los usuarios permisos de lectura y ejecución de los subdirectorios de un directorio:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:33]:
```suggestion
- Recursivamente da a todos [a] los usuarios permisos de lectu[r]a y ejecución [X] de los subdirectorios de un directorio:
```

File: pages.es/linux/chmod.md [hunk 10]
```diff
@@ -0,0 +1,36 @@
...
+
+- Remueve todos los permisos de [o]tros:
+
+`chmod o= {{ruta/al/archivo}}`
+
+- Cambia los permisos recursivamente dando al [g]rupo y a [o]tros la habilidad de escribir [w]:
+
+`chmod -R g+w,o+w {{ruta/al/directorio}}`
+
+- Recursivamente da a todos [a] los usuarios permisos de lectu[r]a y ejecución [x] de los subdirectorios de un directorio:
```
Comment (navarroaxel) [inline pages.es/linux/chmod.md:34]:
```suggestion
- Recursivamente da a todos [a] los usuarios permisos de lectu[r]a y ejecución [X] de los subdirectorios de un directorio:
```

**Review by navarroaxel — APPROVED** [review]

**Review by kbdharun — APPROVED** [review]

**Pull request merged**